src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect, useMemo } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../Loader/Loader';
+import { getPeople } from '../../api';
+import { PersonPage } from '../PersonPage/PersonPage';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isPeopleLoading, setIsPeopleLoading] = useState(false);
+  const [isPeopleError, setIsPeopleError] = useState(false);
+
+  useEffect(() => {
+    setIsPeopleLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setIsPeopleError(true))
+      .finally(() => setIsPeopleLoading(false));
+  }, []);
+
+  const tableHeaders = useMemo(() => [
+    { id: 1, title: 'Name' },
+    { id: 2, title: 'Sex' },
+    { id: 3, title: 'Born' },
+    { id: 4, title: 'Died' },
+    { id: 5, title: 'Mother' },
+    { id: 6, title: 'Father' },
+  ], []);
+
+  const isNoPeople = useMemo(() => (!isPeopleLoading && !people.length),
+    [isPeopleLoading, people]);
```

Comment:
The problem is in the condition checking, something is wrong here. So debug it with different conditions.

Reply:
Thank you so much!